Sources/SpeziLocalStorage/LocalStorage.swift

```diff
@@ -18,10 +18,10 @@ import SpeziSecureStorage
 /// The module relies on the [`SecureStorage`](https://swiftpackageindex.com/StanfordSpezi/SpeziStorage/documentation/spezisecurestorage)
 /// module to enable an encrypted on-disk storage as defined by the ``LocalStorageSetting`` configuration.
 ///
-/// Use ``LocalStorage/store(_:storageKey:settings:)`` to store elements on disk and define the settings using a ``LocalStorageSetting`` instance.
+/// Use ``store(_:encoder:storageKey:settings:)`` to store elements on disk and define the settings using a `LocalStorageSetting` instance.
 ///
-/// Use ``LocalStorage/read(_:storageKey:settings:)`` to read elements on disk which are decoded as define by  passed in  ``LocalStorageSetting`` instance.
-public final class LocalStorage: Module, DefaultInitializable, EnvironmentAccessible {
+/// Use ``read(_:decoder:storageKey:settings:)`` to read elements on disk which are decoded as define by  passed in  `LocalStorageSetting` instance.
+public final class LocalStorage: Module, DefaultInitializable, EnvironmentAccessible, @unchecked Sendable {
```

Comment:
Do you know if there is a fix in Swift 6 to address property wrappers around sensibility or something that we could do in Spezi to address `Stored property '_secureStorage' of 'Sendable'-conforming class 'LocalStorage' is mutable; this is an error in the Swift 6 language mode` to make this a non-unchecked (checked) sendable conformance?

Reply:
Nope sadly not. The only other way I found to have types with property wrappers conform to `Sendable` without requiring `@unchecked Sendable` is by implementing a struct instead of a class avoiding the problem that the underlying storage property might be modified. I hope this is something they address in the future.